test/apiv2/20-containers.at

```diff
@@ -51,6 +51,13 @@ cid=$(jq -r '.[0].Id' <<<"$output")
 
 t DELETE libpod/containers/$cid 204
 
+# Ensure that API does not occur: Create Container creates an invalid and the container fails to start
+# https://github.com/containers/libpod/issues/6799
+CNAME=testArgs
+t POST libpod/containers/create?name=${CNAME} '{"Image":"'"$IMAGE"'"}'}' 200
```

Comment:
The tests take care of converting to JSON. The correct syntax here is:
```
t POST ... Image=$IMAGE
```
Also, the expected return status is 201, not 200

Also, please don't just check return code: add a check for reasonable status. Something like
```
.Id~[0-9a-f]\\{64\\}
```
Also, the `/start` isn't working, I'll let you figure that out.

See `test/apiv2/README.md` for instructions on how to run tests in your local environment.